models/tts/base/tts_dataset.py

```diff
@@ -209,6 +209,9 @@ def __init__(self, cfg, dataset, is_valid=False):
                     phon_id_collator = phoneIDCollation(cfg, dataset=dataset)
                     sequence = phon_id_collator.get_phone_id_sequence(cfg, phones_seq)
 
+                    if cfg.preprocess.add_blank and cfg.train.multi_speaker_training:
```

Comment:
The condition `cfg.train.multi_speaker_training` can be removed, allowing the `intersperse` function to also be applicable to single-speaker training.